projects/addon-doc/languages/russian.ts

```diff
@@ -8,7 +7,7 @@ import {
     TUI_DOC_SEARCH_TEXT,
     TUI_DOC_SEE_ALSO_TEXT,
     TUI_DOC_SOURCE_CODE_TEXT,
-} from '../tokens/i18n';
+} from '@taiga-ui/addon-doc/tokens';
 
 export const TUI_DOC_RUSSIAN: Provider[] = [
```

Comment:
Could you deprecate this entity and add `TODO: drop in v4.0`, please ?

Reply:
fixed